src/models/Text.js

```diff
@@ -1,4 +1,3 @@
-import Collection      from '../core/Collection.js';
```

Comment:
@dwhieb This line was no longer needed bc Utterances are now initiated using ModelDefineProp on line 22, and this file doesn't use any Collection functions, right?

Reply:
Precisely